src/Fabulous/View.fs

```diff
@@ -0,0 +1,41 @@
+namespace Fabulous
+
+module ViewHelpers =
+    let canReuseView (prevWidget: Widget) (currWidget: Widget) =
+        let prevKey = prevWidget.Key
+
+        if not(prevKey = currWidget.Key) then
+            false
+        else if (prevKey = Memo.MemoWidgetKey) then
+            Memo.canReuseMemoizedViewNode prevWidget currWidget
+        else
+            true
+
+module View =
+    let memo<'msg, 'key, 'marker when 'key: equality>
+        (key: 'key)
+        (fn: 'key -> WidgetBuilder<'msg, 'marker>)
+        : WidgetBuilder<'msg, Memo.Memoized<'marker>> =
+
+        let memo: Memo.MemoData =
```

Comment:
Since we're semantically close to Elm like you mentioned the other on Discord, maybe we should adopt the same names? `View.lazy`?

Reply:
Hm, I totally blanked on that fact!!!!! good catch, yes let's rename it. 

If nothing else then we can piggy back on Elm documentation/articles. 

Let's rename it to `View.lazy`, But I think we can leave `Memo` module and `Memo.Memized<'marker>` type as is. Potentially rename `Memo` into `Lazy` (and `Lazy.Memoized<'marker>`), but I don't have a strong opinion on module name

Reply:
bummer, `lazy` is a keyword :( 
<img width="486" alt="image" src="https://user-images.githubusercontent.com/6633216/148636028-dad777d8-78a8-4038-b69c-b0f390322184.png">

Reply:
Ok, let's keep it to `memo` then. 😞